client-network/src/network_ops.rs

```diff
@@ -32,6 +43,7 @@ pub trait NetworkOpsClient {
         from_address: &ExtendedAddr,
         value: Coin,
         attributes: StakedStateOpAttributes,
+        nonce: Nonce,
```

Comment:
I don't know if we want to take `nonce` as input or calculate it while creating a transaction (using `get_staked_state_account()`). Any thoughts @tomtau?

Reply:
nonce is fetched from core.
so it can be contained, but api has two round trips from core.

Reply:
Why will it have two round trips?